make/tools.mk

```diff
@@ -109,6 +109,10 @@ ifdef WINDOWS
 	$(V1) ./downloads/qt-opensource-windows-x86-mingw492-5.5.0.exe
 endif
 
+	# Configure Qt Creator user project files (GCS and flight firmware)
```

Comment:
> well you might want want to clobber your configuration as part of installing the SDK. what if you had customized it and this overwrote it?

I think this points to why I should be using the `.pro.shared` files so we don't ever clobber someone's local customizations. However, since the configuration is, in a sense, an integral part of setting up the Qt Creator IDE, I err toward keeping stuff one-stop-shopping. Introducing a second step required for people to set up the dev environment can lead to a less complete experience, as well as support issues if people have diverging environments.

I guess what I'm arguing for is that the IDE setup is an integral part of the SDK setup. I think you're arguing that many devs/users will not want the (optional) Qt IDE as part of the (required) `qmake` and Qt Libraries. I can certainly see it from that side as well.

I propose I create the separate target, insulated as you suggest, and then if after a bit of testing we can show that it behaves nicely with customized configurations, consider adding it to the SDK.

P.S. I added unit testing and the ability to run `sim_posix` directly from Qt Creator as well. It's been super-useful for improving my iteration time as I run down bugs, but unless you have a preference to include them in the default configuration I feel it's a little clearer leaving them out.